Extension/src/LanguageServer/referencesTreeDataProvider.ts

```diff
@@ -121,7 +121,7 @@ export class ReferencesTreeDataProvider implements vscode.TreeDataProvider<TreeN
                 if (!element.referenceText) {
```

Comment:
empty string will also evaluate to false. Is that something else we should be checking.

Reply:
I don't think we have a lot of cases in which a blank string would be valid.  I addressed some potential cases.